github/activity.go

```diff
@@ -12,3 +12,59 @@ package github
 type ActivityService struct {
 	client *Client
 }
+
+// FeedLink represents a HAL-style link.
```

Comment:
No worries!  I'm learning a bunch here... very interesting.
Should I leave the hal_specification.html link, move it, or just remove it entirely?